serial.go

```diff
@@ -84,12 +86,13 @@ type Config struct {
 	// XONFlowControl bool
 
 	// CRLFTranslate bool
-	// TimeoutStuff int
+	NonBlockingRead bool
+	ReadTimeout     uint32
```

Comment:
This should be a time.Duration instead of a uint32 (see my comment in the README.md)